src/test/java/rx/internal/operators/OnSubscribeRefCountTest.java

```diff
@@ -572,6 +572,11 @@ public void call(Throwable t1) {
                     public void call(String t1) {
                         System.out.println("Subscriber 1: " + t1);
                     }
+                }, new Action1<Throwable>() {
+                    @Override
+                    public void call(Throwable t) {
+                        // ignored
```

Comment:
Same as above, swallowing errors without assert in tests may lead to undetected bugs in future.